test/system/reports_test.rb

```diff
@@ -17,7 +17,9 @@ class ReportsTest < ApplicationSystemTestCase
     assert_selector 'h1', text: '日報'
 
     assert_text 'Railsでテストを書く'
-    assert_text 'alice'
+    within(:xpath, '//table/tbody/tr[1]/td[2]') do
```

Comment:
xpathを使うと可読性が悪くなるので僕はあまり使わないですね。
間違いが発生しない範囲でざっくり範囲を制限すればいいので、今回はいったんこれぐらいでいいかもしれません。

```suggestion
    # tableタグ内に限定する
    within 'table' do
```

実務ではtableタグもページ内に複数あるケースがあるので、もうちょっと工夫が必要になりますが、このプラクティスではこの程度でも十分かなと思いますー。

あと、ついでに `assert_text 'Railsでテストを書く'` もtableタグ内にあると思うので一緒にwithinの中に入れてあげるといいかもしれません